packages/backend/package.json

```diff
@@ -37,6 +37,7 @@
     "@backstage/plugin-catalog-backend": "^0.8.0",
```

Comment:
A general note that above here I expected to see changeset entries for the pre-existing packages that were affected by the pull request, so that they get version bumped and released.